init.sql

```diff
@@ -159,3 +159,128 @@ create table asset_creators
+    frozen_count             int,                                      
+    mint_start               int,
+    freeze_time              int,                                     
+    freeze_fee               int,                                      
+);
+
+create table candy_machine_data
+(
+    id                         bigserial        PRIMARY KEY,
+    uuid                       varchar(MAX),
+    price                      int,
+    symbol                     varchar(5)       not null,
+    seller_fee_basis_points    int              not null,
+    max_supply                 int              not null,
+    is_mutable                 bool             not null,
+    retain_authority           bool,
+    go_live_date               int,
+    items_available            int              not null,
+    candy_machine_id           bytea references candy_machine (id),
+    mode                       whitelist_mint_mode,                                      
+    whitelist_mint             bytea,                                                    
+    presale                    bool,                                                    
+    discount_price             int,
+    prefix_name                varchar(10),                             
+    name_length                int,                                     
+    prefix_uri                 varchar(10),                            
+    uri_length                 int,                                    
+    is_sequential              bool,
+    gatekeeper_network         bytea,                                    
+    expire_on_use              bool,  
+    number                     int,                                     
+    end_setting_type           end_setting_type,    
+    name                       varchar(50),                              
+    uri                        varchar(200),                             
+    hash                       bytea                                                                    
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
+(
+    id                    bigserial                                PRIMARY KEY,
+    candy_machine_id      bytea references candy_machine (id)      not null,
+    creator               bytea                                    not null,
+    share                 int                                      not null default 0,
+    verified              bool                                     not null default false
+);
+create unique index candy_machine_creators_candy_machine_id on candy_machine_creators (candy_machine_id);
+create index candy_machine_creator on candy_machine_creators (candy_machine_id, creator);
+create index candy_machine_verified_creator on candy_machine_creators (candy_machine_id, verified);
+
+create table candy_guard
+(   
+    id                   bytea                                   PRIMARY KEY,
```

Comment:
From our offline conversation, just noting that this ID represents the `base` Pubkey in the `CandyGuard` struct, and that there still needs to be a general solution for knowing how to link the Candy Guard to the specific Candy Machine.

Reply:
yes